lib/profile/info.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+ 
+// TODO: Update this placeholder widget
+class Info extends StatelessWidget {  
```

Comment:
You just created these as placeholders, right? I didn't see any navigations to the placeholder screens when one of the `section_icons` are clicked.

Reply:
Yes, they are just placeholder files right now. I still have to work on the `onPress` methods for the section icons.

Reply:
^ Should come in another PR  soon.

Reply:
👍